hawkbit-ui/src/main/resources/VAADIN/themes/hawkbit/customstyles/generic-styles.scss

```diff
@@ -91,6 +91,26 @@ $generic-text-font-size: $generic-text-font-scale * $v-font-size;
     -webkit-text-overflow: ellipsis;
   }
 
+  .text-bold {
+    font-weight: bold;
+  }
+
+  .header-caption-right {
+    padding-right: 0.5em;
+  }
+
+  .avoid-tooltip {
+    pointer-events: none !important;
+  }
+
+  .v-assistive-device-only {
```

Comment:
why do you need it for, could you provide a comment here?

Reply:
This was needed to make the split of prefix "Artifact details of" and the actual value "smName:smVersion" possible after changing ContentMode from HTML to TEXT. As here are no Labels allowed at the caption field of the artifact details window , I decided to use the methods Window.setAssistivePrefix() and Window.setAssistivePostfix() to solve the XSS-problem. Both, assistive postfix and prefix are usually not visible on the page and only for assistive purpose, but with overwriting this third party html class it's a possible workaround for the XSS vulnerability. I added a comment in the generic-styles.scss.